src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c

```diff
@@ -48,6 +48,8 @@
 		DRM_INFO("packet(0x%p) data[%d] = 0x%x\n",data,i,data[i]); \
 })
 
+//#define SCHED_VERBOSE

```

Comment:
Let's not have this macro in two places.   I don't care if it is on line 26 or 51, but remove one, please.

Reply:
Resolved in my updated PR